server/applicationPattern/applicationPattern/onfModel/models/layerProtocols/HttpClientInterface.js

```diff
@@ -13,9 +13,10 @@ const controlConstruct = require('../ControlConstruct');
 const logicalTerminationPoint = require('../LogicalTerminationPoint');
 const layerProtocol = require('../LayerProtocol');
 const onfPaths = require('../../constants/OnfPaths');
+const ForwardingConstruct = require('onf-core-model-ap/applicationPattern/onfModel/models/ForwardingConstruct');
```

Comment:
do not use package name of current application but use relative paths for imports..also a line below

Reply:
Thank you for Indicating .